lib/board.js

```diff
@@ -0,0 +1,73 @@
+const Line = require('./line.js');
+const Cell = require('./cell.js');
+
+module.exports = class Board {
+  constructor() {
+    this.grid = [
+      new Cell(), new Cell(), new Cell(),
+      new Cell(), new Cell(), new Cell(),
+      new Cell(), new Cell(), new Cell()
+    ]
+  }
+
+  isEmpty() {
+    return this.grid.every(cell => cell.isAvailable());
+  }
+
+  markCellAtPosition(position, symbol) {
+    this.getCellAtPosition(position).markWithSymbol(symbol);
+  }
+
+  getCellAtPosition(position) {
+    return this.grid[position - 1];
+  }
+
+  hasTie() {
+    return (this.isFull() && !this.hasWin());
```

Comment:
Any particular reason to put brackets here?

Reply:
I thought they would be needed, but turns out not :)